dashboard/modules/job/sdk.py

```diff
@@ -104,6 +106,18 @@ def __init__(
             "running Ray 1.9 or higher.",
         )
 
+        # In ray>=2.0, the client sends the new kwarg `submission_id` to the server
+        # upon every job submission, which causes servers with ray<2.0 to error.
+        if packaging.version.parse(self._client_ray_version) > packaging.version.parse(
+            "2.0"
+        ):
+            self._check_connection_and_version(
+                min_version="2.0",
+                version_error_message=f"Client Ray version {self._client_ray_version} "
+                "is not compatible with the Ray cluster. Please ensure the cluster is "
+                "running Ray 2.0 or higher or downgrade the client Ray version.",
+            )
```

Comment:
is this relevant to the resource request change or just a fly-by fix?

Reply:
Ah, the latter, it's just for the `submission_id` change which happened a while ago.  Ran into this during the UX assessment